BraveWallet/Crypto/Search/AssetSearchView.swift

```diff
@@ -47,7 +47,7 @@ struct AssetSearchView: View {
     }
     .navigationViewStyle(StackNavigationViewStyle())
     .onAppear {
-      cryptoStore.tokenRegistry.allTokens { tokens in
+      cryptoStore.blockchainRegistry.allTokens(BraveWallet.MainnetChainId) { tokens in
```

Comment:
Lol I should change the generation so that it generates camel-case constants (i.e. `MainnetChainId` → `mainnetChainId`)